payment-circle/src/main/java/org/stellar/anchor/paymentservice/circle/config/StellarxPaymentConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.stellar.anchor.paymentservice.circle.config;
+
+/**
+ * This is a placeholder class. TODO: move to payment-stellarx subproject under
+ * org.stellar.anchor.paymentservice.circle.config package.
+ */
+public interface StellarxPaymentConfig {
```

Comment:
Same here: Stellarx should be `Stellar`. StellarX is this other product that uses Stellar: https://www.stellarx.com/